style-sheets/primevue-components-overrides.scss

```diff
@@ -271,5 +296,60 @@ label {
 }
 
 /* ------------------------------------ primevue card--------------------------------- */
+.#{variables.$primevue-prefix}-card {
+    box-shadow: none;
+    border: 1px solid colors.$gray-30;
```

Comment:
let's check this color as well

Reply:
Thanks for spotting it, I could find a color for light-theme.